src/views/studio/MyUploadsView/AssetsGroupUploadBar/AssetsGroupUploadBar.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useRef } from 'react'
+import {
+  Container,
+  AssetsGroupUploadBarContainer,
+  ProgressBar,
+  Thumbnail,
+  AssetsInfoContainer,
+  UploadInfoContainer,
+  AssetsDrawerContainer,
+} from './AssetsGroupUploadBar.style'
+import { AssetLine } from '../AssetLine'
+import { Icon, Text } from '@/shared/components'
+import { DrawerButton } from '@/components/Topbar/StudioTopbar/StudioTopbar.style'
+
+type AssetType = 'avatar' | 'cover' | 'thumbnail' | 'video'
+type AssetStatus = 'completed' | 'uploading' | 'pending' | 'failed' | 'reconnecting'
+export type Asset = {
+  id: string
+  type: AssetType
+  progress: number
+  status: AssetStatus
+  width: number
+  height: number
+  size: number
+}
+
+type UploadDataType = 'channel' | 'video'
+export type UploadData = {
+  type: UploadDataType
+  title?: string
+  files: Asset[]
+}
+
+export type AssetsGroupBarUploadProps = {
+  uploadData: UploadData
+}
+
+const AssetsGroupUploadBar: React.FC<AssetsGroupBarUploadProps> = ({ uploadData: { type, title, files } }) => {
+  const [isAssetsDrawerActive, setAssetsDrawerActive] = useState(false)
+
+  const drawer = useRef<HTMLDivElement>(null)
+
+  const isChannelType = type === 'channel'
+  const isPending = files.every((file) => file.status === 'pending')
+  const hasErrorNumber = files.filter(({ status }) => status === 'failed' || status === 'reconnecting').length
+
+  const allAssetsSize = files.reduce((acc, file) => (acc = acc + file.size), 0)
+  const alreadyUploadedSize = files.reduce((acc, file) => (acc = acc + (file.size * file.progress) / 100), 0)
```

Comment:
You don't need to assign to `acc` here, only thing that matters is the return value
```suggestion
  const allAssetsSize = files.reduce((acc, file) => acc + file.size, 0)
  const alreadyUploadedSize = files.reduce((acc, file) => acc + (file.size * file.progress) / 100, 0)
```